packages/react/src/components/DataTable/TableToolbarSearch.js

```diff
@@ -96,6 +98,9 @@ const TableToolbarSearch = ({
     }
   };
 
+  const internalOnFocus = (event) => handleExpand(event, true);
```

Comment:
```suggestion
  const handleOnFocus = (event) => handleExpand(event, true);
```

We are working towards a standard of internal functions using `handleOn...()` with the prop for that functionality being `on...()`

Reply:
Thanks, @abbeyhrt. Updated.